src/services/findAllReferences.ts

```diff
@@ -1425,6 +1429,20 @@ namespace ts.FindAllReferences {
             }
         }
 
+        function getAllReferencesForImportMeta(sourceFiles: readonly SourceFile[], cancellationToken: CancellationToken): SymbolAndEntries[] | undefined {
+            const references = flatMap(sourceFiles, sourceFile => {
+                cancellationToken.throwIfCancellationRequested();
+                return mapDefined(getPossibleSymbolReferenceNodes(sourceFile, "import.meta", sourceFile), node => {

```

Comment:
This doesn't work in the presence of whitespace.

```ts
let x = import
  .  // hai :)
    meta;
```

I would change your search string to `meta` instead.